npc/re/jobs/novice/academy.txt

```diff
@@ -13,75 +13,241 @@
 //= 1.0.0 Initial Release. [Kisuka]
 //= 1.0.1 Converted item constants to IDs and added hair coupon. [Kisuka]
 //= 2.0.0 Converted from Hercules Script to rAthena Script. [Capuche]
+//= 2.0.1 Add new Intro Renewal based on Hercules [Jenkijo]
 //==============================================================================
 
+//== Starting Point  (iz_int) =================================
+iz_int,18,26,0	script	startpoint	HIDDEN_WARP_NPC,1,1,{
+OnTouch:
+	if (!izintspawn) {
+		navigateto("int_land", NAV_NONE, 1); // individual map name not required
+		izintspawn = 1;
+	}
+	end;
+}
+
+//== Wounded Swordsman (iz_int) =================================
+iz_int,56,32,3	script	Wounded Swordsman#izint	4_TOWER_01,8,8,{
+	if (!@display) {
+		classchange 4_TOWER_02, "", bc_self;
+		npctalk "Ohhhhhhh....", "", bc_self;
+		@display = 1;
+		end;
+	}
+	switch( isbegin_quest(21001) ) {
+	case 0:
+		mes "[Wounded]";
+		mes "Wow! Thanks a lot!";
+		mes "I don't know how this happened to our ship";
+		mes "but we should go to see the captain.";
+		next;
+		mes "[Wounded]";
+		mes "... ohh, it seems my body is injured too much.";
+		mes "Maybe you can go without me?";
+		setquest 21001;
+		next;
+		break;
+	case 1:
+		mes "[Wounded]";
+		mes "Luckily the water has not entered the ship.";
+		mes "I will wait here, you should go see the captain.";
+		next;
+		mes "[Wounded]";
+		mes "I'm fine! I'm fine!";
+		mes "Don't worry about me, you should really go to the captain!";
+		next;
+		break;
+	case 2:
+		break;
+	}
+	cutin "tutorial02",4;
+	mes "^4d4dff!- Information -!^000000";
+	mes "NPC Quest Received.";
+	mes "^4d4dffQuestinfo Shortcut is Alt + U^000000 You can check your quest status there anytime.";
+	close2;
+	cutin "",255;
+	end;
+
+OnInit:
+	questinfo 21001, QTYPE_QUEST, 1, Job_Novice;
+	end;
+
+OnTouch_:
+	if (@cd + 15 > gettimetick(2))
```

Comment:
This check is stupid. The real check should be like this:
```
OnTouch_:
	if( @cd ){
		// NPC is not finished yet
		end;
	}
	@cd = 1;
	npctalk "Urgh... please... help me...", "", bc_self;
	sleep2 7000;
	npctalk "In order to talk with me.. click my body..", "", bc_self;
	@cd = 0;
	end;
```
He does not have any cooldown at all, it's just about reentering the ontouch area and if he is finished talking yet.
The delay in between is a lot shorter than 7 seconds as well. It's about half a seconds(500ms probably).
![grafik](https://user-images.githubusercontent.com/3517879/29195010-a078c09c-7e2c-11e7-9cb9-5e6d74c36dd6.png)